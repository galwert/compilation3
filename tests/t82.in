void printOk(bool isOk) {
    if (isOk) print("ok");
}

void printInt(int x) {
    printi(x);
}

void main()
{
    //these are ok :)
    printInt(10 if (5>3) else 2);
    printOk(true if (5>3) else false);
    
    // this is bad - missing () after if
    print("ok" if 5>3 else "bad");
}