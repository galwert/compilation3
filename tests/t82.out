---end scope---
---end scope---
isOk BOOL -1
---end scope---
x INT -1
line 16: syntax error
